server/container/guice/lucene/src/main/java/org/apache/james/modules/mailbox/LuceneMemorySearchMailboxModule.java

```diff
@@ -21,15 +21,24 @@
 
 import java.io.IOException;
 
+import org.apache.james.mailbox.lucene.search.LuceneSearchHighlighter;
+import org.apache.james.mailbox.searchhighligt.SearchHighlighter;
 import org.apache.lucene.store.ByteBuffersDirectory;
 import org.apache.lucene.store.Directory;
 
 import com.google.inject.AbstractModule;
 import com.google.inject.Provides;
+import com.google.inject.Scopes;
 import com.google.inject.Singleton;
 
 public class LuceneMemorySearchMailboxModule extends AbstractModule {
 
+    @Override
+    protected void configure() {
+        bind(SearchHighlighter.class).to(LuceneSearchHighlighter.class)
```

Comment:
Is this specific to the memory implem? Or shall it land onto the LuceneBlahBlahModule?